lib/js/jsffi.nim

```diff
@@ -184,6 +184,7 @@ proc `and`*(x, y: JsObject): JsObject {.importcpp: "(# && #)".}
 proc `or` *(x, y: JsObject): JsObject {.importcpp: "(# || #)".}
 proc `not`*(x:    JsObject): JsObject {.importcpp: "(!#)".}
 proc `in` *(x, y: JsObject): JsObject {.importcpp: "(# in #)".}
+proc `**` *(x, y: JsObject): JsObject {.importcpp: "(# ** #)".} ## https://developer.mozilla.org/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators#Exponentiation
```

Comment:
The link makes the line too long. Put it on a different line or remove it entirely, I am not sure if we need it.